src/components/ListSelector/styles.module.css

```diff
@@ -31,9 +31,10 @@
     background: transparent;
 }
 
-.listSelector li {
+.listSelectorItem {
     padding: 6px 26px;
     border-radius: 20px;
+    border-width: 0;
```

Comment:
I would add in "outline: 0" here too, so the outline doesn't appear when you select a list button